api/controllers/load_balancer.go

```diff
@@ -78,3 +79,19 @@ func (l *LoadBalancerController) ListLoadBalancers(c *fireball.Context) (firebal
 	return fireball.NewJSONResponse(200, summaries)
 
 }
+
+func (l *LoadBalancerController) UpdateLoadBalancer(c *fireball.Context) (fireball.Response, error) {
+	id := c.PathVariables["id"]
+
+	var req models.UpdateLoadBalancerRequest
+	if err := json.NewDecoder(c.Request.Body).Decode(&req); err != nil {
+		return nil, errors.New(errors.InvalidRequest, err)
+	}
+
+	req.LoadBalancerID = id
```

Comment:
This behavior is a little odd. Since we need the `LoadBalancerID` field in the `UpdateLoadBalancerRequest` model for it to be properly handled as a job, let's move  this request to `PUT /loadbalancer`

Reply:
:+1: Handler is now PUT against /loadbalancer